system-test/spanner.ts

```diff
@@ -4405,6 +4405,31 @@ async function deleteTestInstances() {
   );
 }
 
+async function deleteOldTestInstances() {
+  const [instances] = await spanner.getInstances();
+  // Leave only instances that contain PREFIX in their name
+  // and where created more that an hour ago.
+  const toDelete = instances.filter(
+    instance =>
+      instance.id.includes(PREFIX) &&
+      (Math.round(Date.now() / 1000) -
```

Comment:
Could this be moved out and coupled with some logical variable naming so that it's easier to figure out what's going on? Feels a bit magical right now.

Reply:
Refactored calculations as method with meaningful name